runtime/version_info.json

```diff
@@ -0,0 +1,3 @@
+{
+  "package-version": "3.0.0.dev"
+}
```

Comment:
Seems like we could have `setup.py` insert the `.dev` or `.rc` suffix as needed. What would make the most sense to have in the source code itself?

Reply:
If we do, how to tell `setup.py` if to build a dev release, a nightly release or something else? Additional arguments? Environment variables?